pom.xml

```diff
@@ -22,7 +22,7 @@
     <groupId>com.akathist.maven.plugins.launch4j</groupId>
     <artifactId>launch4j-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>2.3.0-SNAPSHOT</version>
+    <version>2.3.0</version>
```

Comment:
This doesn't work like that - I can release a new version using Maven Release plugin and pushing into Sonatype OSS repository (then it will be replicated to the Maven Central)

Reply:
Ok, so should I left version `2.3.0-SNAPSHOT` as it is? Then you will cover it manually via Sonatype?

Reply:
Yes, please revert to the `-SNASPHOT` version.

The whole release process is handled by [the Maven Release plugin](https://maven.apache.org/maven-release/maven-release-plugin/), first you must have an account in Sonatype Nexus (which takes time and so, so I will do it), second you use the [release:prepare](https://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html) mojo to setup everything (like validating, trimming -SNASPHOT, tagging repo, etc), finally you use the [release:perform](https://maven.apache.org/maven-release/maven-release-plugin/perform-mojo.html) mojo to perform the release process (the tag is checked out, build and pushed into OSS Sonatype Nexus) - all these happens semi-automatic, you just need to start the process ;-)

Reply:
Thank you very much for the explanation of the release process!
I've reverted to the `-SNAPSHOT` version.